docs/root/intro/arch_overview/http_connection_management.rst

```diff
@@ -60,6 +60,9 @@ can be used to modify this behavior, and they fall into two categories:
   * *envoy.retry_host_predicates.previous_hosts*: This will keep track of previously attempted hosts, and rejects
     hosts that have already been attempted.
 
+  * *envoy.retry_host_predicates.omit_canary_hosts*: This will ensure that a retry will always be done on
```

Comment:
I would change the wording here since the host predicate doesn't *guarantee* that a non-canary host, it just keeps picking a new host until one satisfies the requirement (or the max attempt count has been reached). For example, if all hosts are marked canary it will still select a canary host.

Reply:
Agree - That is not well worded. How about this:

> This will reject any host that is a marked as canary host.

I think we could also add a note on how canary hosts / endpoints are marked and a link to the `endpoint.LbEndpoint` docs:

>  Hosts are marked by setting `canary: true` for the `envoy.lb` filter in the endpoint's filter metadata. See :ref:`LbEndpoint<envoy_api_msg_endpoint.LbEndpoint>` for more details.

Reply:
Yeah, that sounds great to me.